eth/stagedsync/stage_execute.go

```diff
@@ -35,12 +38,13 @@ type StateReaderBuilder func(ethdb.Getter) state.StateReader
 type StateWriterBuilder func(db ethdb.Database, changeSetsDB ethdb.Database, blockNumber uint64) state.WriterWithChangeSets
 
 type ExecuteBlockStageParams struct {
-	ToBlock       uint64 // not setting this params means no limit
-	WriteReceipts bool
-	BatchSize     int
-	ChangeSetHook ChangeSetHook
-	ReaderBuilder StateReaderBuilder
-	WriterBuilder StateWriterBuilder
+	ToBlock               uint64 // not setting this params means no limit
+	WriteReceipts         bool
+	BatchSize             int
+	ChangeSetHook         ChangeSetHook
+	ReaderBuilder         StateReaderBuilder
+	WriterBuilder         StateWriterBuilder
+	SilkwormExecutionFunc unsafe.Pointer
```

Comment:
Any profit from using unsafe.Pointer vs uintptr?

Reply:
Well, it's really a pointer and can be nil, why make it an integer.